connexion/exceptions.py

```diff
@@ -95,3 +96,20 @@ def __init__(self, token_scopes, required_scopes, title=None, **kwargs):
         self.missing_scopes = required_scopes - token_scopes
 
         super(OAuthScopeProblem, self).__init__(title=title, **kwargs)
+
+
+class ExtraParameterProblem(ProblemException, BadRequest):
+    def __init__(self, formdata_parameters, query_parameters, title=None, description=None, **kwargs):
```

Comment:
Correct. Or you could have something like:

```
{
    "type": "https://example.net/validation-error",
    "title": "Your request has extra parameters in the query.",
    "extra_params": ["name", "age"],
}
```
Or like you can keep b/c with the existing code (like i have with the description)

The point is having all this information in the exception allows you the choice :) You're giving the user as much information as possible to allow them to render the exception into the format they decide is most optimal for their use case.